druid/src/core.rs

```diff
@@ -968,6 +976,11 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                     if let Some(change) = this_changed {
                         self.state.has_focus = change;
                         extra_event = Some(LifeCycle::FocusChanged(change));
+
+                        if change {
+                            //TODO: decide whether this should be done manually
```

Comment:
I vote for manual

Reply:
I couldn't come up with a case, where you don't want to do it. It feels natural to display the focused widget, but I don't have a strong opinion on this